HTML/styles.scss

```diff
@@ -0,0 +1,53 @@
+.level1 {
+    padding: 10px;
+    background-color: #f0f0f0;
+
+    .level2 {
+        padding: 10px;
+        background-color: #d0d0d0;
+
+        .level3 {
+            padding: 10p;
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/17#issuecomment-2189295790>#b83368</a><span> - 06/25/2024, 03:44 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The padding value for the '.level3' class is incorrectly specified as '10p', which is not a valid CSS unit and will cause the property to be ignored.
</div>
<div id="fix">
<b>Fix:</b> Correct the padding value to use a valid CSS unit, such as '10px'.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -10,7 +10,7 @@
      .level3 {
 -        padding: 10p;
 +        padding: 10px;
          background-color: #b0b0b0;
      }
```
</div>
</details>
</div>
<br>


</div>